.github/workflows/on_push.yaml

```diff
@@ -2,21 +2,23 @@ name: On Push
 on:
   push:
     branches-ignore:
-      - "staging"
-      - "master"
+      - 'staging'
+      - 'master'
 
 jobs:
   build:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v3
+      - uses: actions/checkout@v4
       - name: Install node
-        uses: actions/setup-node@v3
+        uses: actions/setup-node@v4
         with:
-          cache: "yarn"
-          node-version: 16
-          registry-url: "https://registry.npmjs.org"
+          cache: 'yarn'
+          node-version: 18
+          registry-url: 'https://registry.npmjs.org'
       - name: Install deps
         run: yarn
+      - name: Render svg images
+        run: yarn svg-images
```

Comment:
is running this command is always needed locally ?
if yes, it should not be here it should part of yarn build in the package.json

Reply:
no, it is just needed if changes in the drawio.svg images are done (which I assume is almost never)